src/components/SearchJob.jsx

```diff
@@ -2,47 +2,70 @@ import React, { useState } from 'react';
 import {
 	Form,
 } from 'react-bootstrap';
-import cn from 'classnames';
+import styled from 'styled-components';
 
 function SearchJob() {
-	const [open] = useState(false);
+	const StyledForm = styled(Form)`
+		margin-top: 15px;
+		/* border: solid 1px rgb(39, 39, 39); */
+		border-radius: 5px;
+		width: 100%;
+		justify-content: center;
+`;
+
+	const StyledFormControl = styled(Form.Control)`
+		background-color: #292727;
+		color: #d97b16;
+		border-radius: 20px;
+		padding: 6px 20px;
+		border-color: #5c0f0f;
+		width: 100%;
+		outline: none;
+		text-align: center;
+		&:focus {
+		border-color: #5c0f0f;
+		outline: none;
+		color: #e9e9e9;
+		box-shadow: none;
+		background-color: rgb(85, 83, 83);
+}
+`;
+	const StyledFormLabel = styled(Form.Label)`
```

Comment:
Shouldn't be a space here and add a new line above

Reply:
Also should probably be its own component file so you're not copying this in a bunch of places

Reply:
i had done that in the first branch that i asked you merge first.....can you do that so that my dev branch can have same styled stuffs

![image](https://user-images.githubusercontent.com/25382246/106936134-41334380-672d-11eb-9014-b35f890362f9.png)